catalog/ornl-mapspace/projects_ornl-mapspace_assets_grid_3c.jsonnet

```diff
@@ -0,0 +1,127 @@
+local id = 'projects/ornl-mapspace/assets/grid_3c';
+local subdir = 'ornl-mapspace';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+
+
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+local units = import 'units.libsonnet';
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'MapSpace Global Residential/Non_Residential Landuse Model',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: ||| 
+    Global residential/non-residential landuse classification generated based on spatial distribution and semantic attributes of Points of Interest(POI) data.
+      
+    Please refer to the [journal article](https://doi.org/10.1080/17538947.2023.2174607) for technical details about the methodology used to generate the dataset. 
+
+    See [Office Website of MapSpace Program](
+    https://mapspace.ornl.gov) for
+    more details.
+  |||,
+
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference),
+
+  ],
+  keywords: [
+    'landuse',
+    'residential',
+    'points_of_interest',
+    'non_residential',
+    'mapspace',
+    'ornl'
+  ],
+  providers: [
+    ee.producer_provider('ORNL', 'https://mapspace.ornl.gov/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2024-11-01T00:00:00Z', '2025-11-01T00:00:00Z'),
+  summaries: {
+
+    gsd: [500],
+    platform: ['ORNL'],
+ 
+    'eo:bands': [
+      {
+      name: 'label',
```

Comment:
as mentioned to Kel, could you please reingest setting this band's pyramiding policy to MODE?

Reply:
I have reingested the data accordingly.